### DISCUSSION TITLE
New Elyra pipeline viewer for GitHub

### DISCUSSION THREAD

**Discussion by bourdakos1** [elyra-ai/elyra#1208] [Show and tell]
Hey everyone!

I just wanted to show off a Chrome extension I've built to make viewing Elyra pipelines much easier. You can check out the very first version of the extension [here 🎉](https://chrome.google.com/webstore/detail/elyra-pipeline-viewer/blomglkicbbfkbmppeljdnnakajnpnop)

![](https://github.com/elyra-ai/elyra-pipeline-chrome-extension/raw/main/screenshots/demo.png)

For those interested in how it all works, you can find the source code [here](https://github.com/elyra-ai/elyra-pipeline-chrome-extension).

**Comment by ptitzler** [comment]
Love it! I just tried the extension on Chrome/Windows. Works like a charm. Thanks for creating it!

![image](https://user-images.githubusercontent.com/13068832/105065623-e910f780-5a32-11eb-820b-e56b586c2054.png)

**Comment by bourdakos1** [comment]
The extension is also available for firefox now: https://addons.mozilla.org/en-US/firefox/addon/elyra-pipeline-viewer/
